v2/tools/generator/internal/astmodel/known_locals_set.go

```diff
@@ -28,15 +28,44 @@ func NewKnownLocalsSet(idFactory IdentifierFactory) *KnownLocalsSet {
 	}
 }
 
+// flectOverrides is an ordered list of overrides for flect pluralization
+// Earlier overrides take precedence over later ones
+var flectOverrides = []struct {
```

Comment:
There is a similar construct in `type_name.go`, see the `Singularize` function there.

Should we consider combining these into a flect-library wrapper? So instead of this we would do:

`ourflect.Singularize()` or `ourflect.Pluralize()` in both `type_name.go` and here, and we could have a shared set of overrides for all.

Reply:
That's a good idea, but not one I want to pursue in this PR. I'll make a note to do this later.

Reply:
Maybe leave a TODO here?